### PR TITLE
Incremental 112.35.00

### PR DESCRIPTION
**Pull request by trefis**

### CODE REVIEW

**Comment by trefis** [comment]
It apparently passed travis tests, so I updated the PR to point to the final version.
I believe this can be merged.

File: packages/incremental/incremental.112.35.00/opam [hunk 1]
```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/incremental"
+license: "Apache-2.0"
+build: [
+  [make]
+  [make "install"]
```
Comment (avsm) [inline packages/incremental/incremental.112.35.00/opam]: could this `make install` be moved into the `install:` target instead of build?

Comment (avsm) [inline packages/incremental/incremental.112.35.00/opam]: (try `opam lint` to see more)

Comment (trefis) [inline packages/incremental/incremental.112.35.00/opam]: Done, thank you!

File: packages/incremental/incremental.112.35.00/opam [hunk 2]
```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/incremental"
+dev-repo: "https://github.com/janestreet/incremental"
```
Comment (dsheets) [inline packages/incremental/incremental.112.35.00/opam]: need `.git` on end here

**Pull request merged**